docs/breaking.md

```diff
@@ -1,5 +1,406 @@
 - Cancelling a job is no longer a `POST` to `/project/:project-slug/job/:jobnum`. The job cancellation action is now a `POST` to `/project/:project-slug/job/:jobnum/cancel`
+
+# Proposed Breaking Changes
+
+## Oct 21, 2019
+
+The API v2 preview was made available earlier this year with the intent of exposing [new workflow and pipeline control endpoints](https://github.com/CircleCI-Public/api-preview-docs/blob/master/docs/api-changes.md#new-endpoints-available-for-preview-use) to our API users. With the v2 Preview endpoints in place, the team at CircleCI also began to migrate API v1 and v1.1 endpoints into v2.
+
+As we began to migrate endpoints into v2, the team took some time to re-evaluate the design and UX of the overall API itself. It was during this process that we identified some gaps and inconsistencies with how the v2 endpoints behaved. These changes would require us to introduce breaking changes, but contributes to a more consistent and scalable API v2 experience.
+
+The changes we are making fall into one of five general categories:
+
+- Removed endpoints
+- HTTP responses
+- Changes to routes
+- Paginated responses
+- Representation of objects in responses.
+
+### Removed endpoints
+
+The list of removed endpoints have already been announced in the original API v2 preview announcement. You can find the list of endpoints that didn't make the transition from `v1.1` to `v2` [here](https://github.com/CircleCI-Public/api-preview-docs/blob/master/docs/api-changes.md#endpoints-likely-being-removed-in-api-v2-still-available-in-v11-for-now).
+
+There are no other endpoints marked for removal to share at this time.
+
+### Consistent HTTP Responses
+
+We've caught a few situations where some API endpoints don't respond with the expected HTTP Status code. We will be standardizing our HTTP responses to make sure that all our `v2` endpoints provide a consistent HTTP response status.
+
+#### Successful Status Codes
+
+200 OK - responds with a custom payload
+201 Created - responds with the created entity
+202 Accepted -  responds with `{“message”: “Accepted.”}`
+
+#### Error Status Codes
+
+##### User Errors
+
+For user fault errors, the error message will be human readable and have enough information for a developer to understand and address the problem.
+
+###### 404 Not Found
+
+Generally responds with:
+
+```json
+{“message”: “some-entity-name not found.”}
+```
+
+This code will be used when:
+
+- The requested resource is not found
+- The URI is incorrect, including project slug being incorrect, or job number/workflow ID being malformed. This does not include query parameters, which we can parse and validate.
+- Unauthenticated requests to resource
```

Comment:
Can we keep this a bit more generic? Detailed documentation around error codes was more intended for API authors. So maybe something like this: 

Some example when you will see this error code: resource not found, or URI doesn't exist or is malformed.